### PR TITLE
Fix method generation for those who don't have arguments

### PR DESCRIPTION
**Pull request by whatUwant**
Fixes https://github.com/phpro/soap-client/issues/316

### CODE REVIEW

**Comment by whatUwant** [comment]
Any update on this PR? @veewee @janvernieuwe

**Comment by veewee** [comment]
Thanks for the PR @whatUwant. We will review it shortly!

File: src/Phpro/SoapClient/CodeGenerator/Assembler/ClientMethodAssembler.php [hunk 1]
```diff
@@ -42,21 +42,27 @@ public function assemble(ContextInterface $context): bool
             $phpMethodName = Normalizer::normalizeMethodName($method->getMethodName());
             $param = $this->createParamsFromContext($context);
             $class->removeMethod($phpMethodName);
-            $docblock = $context->isMultiArgument() ?
+            $docblock = $context->getArgumentCount() > 1 ?
                 $this->generateMultiArgumentDocblock($context) :
                 $this->generateSingleArgumentDocblock($context);
+            $methodBody = $param === null ?
```
Comment (janvernieuwe) [inline src/Phpro/SoapClient/CodeGenerator/Assembler/ClientMethodAssembler.php:48]: Perhaps move this to a private method with early returns?

**Review by janvernieuwe — CHANGES_REQUESTED** [review]
Hi @whatUwant 

Thank you very much for your PR, it's looking good.

Could you change the big ternary to a private method?
This would improve readability a lot.

**Review by janvernieuwe — APPROVED** [review]

**Comment by janvernieuwe** [comment]
Thank you!

**Pull request merged**